### ISSUE TITLE
[3.0] API Platform + NelmioApiDocBundle prefix issues

### ISSUE THREAD

**Issue by simshaun** [nelmio/NelmioApiDocBundle#1140]
I'm using API Platform + NelmioApiDocBundle.

The API Platform's built-in docs are correct and look like this:
![image](https://user-images.githubusercontent.com/379519/33824698-3d2854ba-de2d-11e7-91e2-4eec8cb51430.png)

And this is what the Nelmio API docs look like. It's adding an extra `/api`.
![image](https://user-images.githubusercontent.com/379519/33824720-50b599d4-de2d-11e7-8aef-e3d4209143c2.png)

My API Platform routing is:

```yaml
api_platform:
    resource: .
    type: api_platform
    prefix: /api
```

**Comment by MJBGO** [comment]
I have the same issue :(

https://github.com/nelmio/NelmioApiDocBundle/issues/1128

**Comment by GuilhemN** [comment]
Not sure what to do here... I have to investigate why this is happening but I do not have much time these days :/
Would greatly appreciate help here :)

**Comment by 7thcubic** [comment]
not sure exactly what is happening but from my understanding.

routing prefix: /foo
controller route: /bar
action route: /action

will generate /foo/bar/action which is correct.

so not entirely sure what was being generated by the api platform bridge, need to investigate

**Comment by GuilhemN** [comment]
Should be fixed by #1172
